src/debian/01_password

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/sh
+
+# Set grub password : "grub-mkpasswd-pbkdf2 -c 65536 -s 256 -l 64" and replace $GRUB_PASS
+# Default password : toor
```

Comment:
Not secure at all since this source code is public :stuck_out_tongue: 
OK for this commit, but add another one after that that takes the grub password from the inventory definition. Also document it.